DfciPkg/UnitTests/DfciTests/Support/Python/DFCI_SupportLib.py

```diff
@@ -9,17 +9,24 @@
 #
 # DFCI_SupportLib
 #
-import os, sys
+import os
 import xml.etree.ElementTree as ET
 import binascii
 import traceback
 import xml.dom.minidom
 import subprocess
+import configparser
+import json
+import struct
+import pathlib
 
-try:
-    from StringIO import StringIO
-except ImportError:
-    from io import StringIO, BytesIO
+
+# try:
+# from StringIO import StringIO
+# except ImportError:
+from io import StringIO, BytesIO
+# from cryptography.hazmat.primitives.asymmetric import rsa
+# from cryptography.hazmat.primitives.serialization import load_der_public_key

```

Comment:
Do we want to keep the commented out code or can we remove it?

Reply:
Fixed